{"content": {"suites": {"0": {"status": {"total_pass": 16, "total_skip": 0, "total_xpass": 0, "total_xfail": 0, "total_rerun": 0, "total_fail": 0, "total_error": 1}, "tests": {"0": {"status": "PASS", "message": "", "test_name": "test_update_password_with_valid_data[testuser1@momenttext.com-Password@1235445323-4yT28GN7]", "rerun": "0"}, "1": {"status": "PASS", "message": "", "test_name": "test_update_password_with_valid_data[testuser2@momenttext.com-Password@123545jkw-4yT28GN7]", "rerun": "0"}, "2": {"status": "PASS", "message": "", "test_name": "test_update_password_with_valid_data[testuser3@momenttext.com-Password@123545jkw-4yT28GN7]", "rerun": "0"}, "3": {"status": "PASS", "message": "", "test_name": "test_update_password_with_valid_data[testuser4@momenttext.com-Password@123545jkw-4yT28GN7]", "rerun": "0"}, "4": {"status": "PASS", "message": "", "test_name": "test_update_password_with_invalid_password_pattern[testuser1@momenttext.com-Password@12354-4yT28GN7]", "rerun": "0"}, "5": {"status": "PASS", "message": "", "test_name": "test_update_password_with_invalid_password_pattern[testuser2@momenttext.com-Password@12354-4yT28GN7]", "rerun": "0"}, "6": {"status": "PASS", "message": "", "test_name": "test_update_password_with_invalid_password_pattern[testuser3@momenttext.com-Password@12354-4yT28GN7]", "rerun": "0"}, "7": {"status": "PASS", "message": "", "test_name": "test_update_password_with_invalid_password_pattern[testuser4@momenttext.com-Password@12354-4yT28GN7]", "rerun": "0"}, "8": {"status": "PASS", "message": "", "test_name": "test_update_password_with_invalid_password_pattern[testuser5@momenttext.com-Password@12354-4yT28GN7]", "rerun": "0"}, "9": {"status": "PASS", "message": "", "test_name": "test_update_password_with_invalid_password_pattern[testuser6@momenttext.com-Password@12354-4yT28GN7]", "rerun": "0"}, "10": {"status": "PASS", "message": "", "test_name": "test_update_password_with_invalid_password_pattern[testuser7@momenttext.com-Password@12354-4yT28GN7]", "rerun": "0"}, "11": {"status": "PASS", "message": "", "test_name": "test_update_password_with_invalid_password_pattern[testuser8@momenttext.com-Password@12354-4yT28GN7]", "rerun": "0"}, "12": {"status": "PASS", "message": "", "test_name": "test_update_password_without_access_management[testuser1@momenttext.com-Password@12354-4yT28GN7]", "rerun": "0"}, "13": {"status": "PASS", "message": "", "test_name": "test_update_password_without_access_management[testuser2@momenttext.com-Password@12354-4yT28GN7]", "rerun": "0"}, "14": {"status": "PASS", "message": "", "test_name": "test_update_password_without_access_management[testuser3@momenttext.com-Password@12354-4yT28GN7]", "rerun": "0"}, "15": {"status": "PASS", "message": "", "test_name": "test_update_password_without_access_management[testuser4@momenttext.com-Password@12354-4yT28GN7]", "rerun": "0"}, "16": {"status": "ERROR", "message": "file D:\\projects\\backend\\backend\\usermanagement\\Test_usermanagement\\test_views.py, line 396\n  \tdef test_update_password_with_missing_data(self,client,email,password,otp,setup_user_without_access):\nE       fixture 'email' not found\n>       available fixtures: _dj_autoclear_mailbox, _django_clear_site_cache, _django_db_marker, _django_set_urlconf, _django_setup_unittest, _fail_for_invalid_template_variable, _live_server_helper, _session_faker, _template_string_if_invalid_marker, admin_client, admin_user, async_client, async_rf, cache, capfd, capfdbinary, caplog, capsys, capsysbinary, celery_app, celery_class_tasks, celery_config, celery_enable_logging, celery_includes, celery_parameters, celery_session_app, celery_session_worker, celery_worker, celery_worker_parameters, celery_worker_pool, client, db, depends_on_current_app, django_assert_max_num_queries, django_assert_num_queries, django_capture_on_commit_callbacks, django_db_blocker, django_db_createdb, django_db_keepdb, django_db_modify_db_settings, django_db_modify_db_settings_parallel_suffix, django_db_modify_db_settings_tox_suffix, django_db_modify_db_settings_xdist_suffix, django_db_reset_sequences, django_db_setup, django_db_use_migrations, django_mail_dnsname, django_mail_patch_dns, django_test_environment, django_user_model, django_username_field, doctest_namespace, enable_db_access_for_all_tests, faker, live_server, mailoutbox, monkeypatch, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, rf, settings, setup_activitylogs, setup_company, setup_invalid_attempt_update, setup_invalid_first_attempt, setup_multiple_invalid, setup_multiple_invalid_for_pass_reset, setup_multiple_invalid_reset, setup_random_user, setup_register_access, setup_register_worker, setup_roles, setup_saved_user, setup_superusers, setup_temporary_urls, setup_update_password, setup_update_password_access_counter, setup_user, setup_user_without_access, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory, transactional_db, use_celery_app_trap\n>       use 'pytest --fixtures [testpath]' for help on them.\n\nD:\\projects\\backend\\backend\\usermanagement\\Test_usermanagement\\test_views.py:396\n", "test_name": "test_update_password_with_missing_data", "rerun": "0"}}, "suite_name": "usermanagement/Test_usermanagement/test_views.py"}}}, "date": "August 19, 2021", "start_time": 1629381802.3313966, "total_suite": 1, "status": "FAIL", "status_list": {"pass": "16", "fail": "0", "skip": "0", "error": "1", "xpass": "0", "xfail": "0", "rerun": "0"}, "total_tests": "17"}